modules/publication/ajax/FileDownload.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Publication Downloader
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Publication
+ * @author   David <dblader.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+$user = \User::singleton();
+
+if (!($user->hasPermission('publication_view')
```

Comment:
There's the new hasAnyPermission function which can simplify this (and similar) checks

Reply:
that's on major, this branch was based off minor